classes/help.php

```diff
@@ -71,64 +70,64 @@ function print_page() {
 								<a name="displaying_your_map"></a>
 								<div class="postbox">
 
-									<h3><?php _e( 'Displaying Your Map', 'SimpleMap' ); // XSS ok. ?></h3>
+									<h3><?php _e( 'Displaying Your Map', 'simplemap' ); // XSS ok. ?></h3>
 
 									<div class="inside" style="padding: 0 10px 10px 10px;">
 
 										<div class="table">
 											<table class="form-table">
 
 												<tr>
-													<td><?php _e( 'To show your map on any post or page, insert the shortcode in the body:', 'SimpleMap' ); // XSS ok. ?>
+													<td><?php _e( 'To show your map on any post or page, insert the shortcode in the body:', 'simplemap' ); // XSS ok. ?>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 169 characters

<!-- Comment by [SideCI](https://sideci.com) -->